.vscode/launch.json

```diff
@@ -57,19 +57,6 @@
             "cwd": "${workspaceFolder}",
             "console": "internalConsole"
         },
-        {
-            "name": "server: chrome",
-            "type": "chrome",
-            "request": "launch",
-            "url": "http://localhost:7000/",
-            "disableNetworkCache":true,
-            "trace": true,
-            "showAsyncStacks": true,
-            "pathMapping":{
-                "/static/engine/": "${workspaceFolder}/cvat/apps/engine/static/engine/",
-                "/static/dashboard/": "${workspaceFolder}/cvat/apps/dashboard/static/dashboard/",
-            }
-        },
```

Comment:
I know that @Marishka17  has used this configuration recently

Reply:
Since we have a proxy to the server on webpack you can use UI - debug task. But I will keep it if any user needs this config.